idris-commands.el

```diff
@@ -238,7 +238,9 @@ line."
            (lambda (_condition)
              (when (member 'warnings-tree idris-warnings-printing)
                (idris-list-compiler-notes)
-               (pop-to-buffer (idris-buffer-name :notes)))))))
+	       (if idris-stay-in-current-window-on-compiler-error
```

Comment:
here's now a tab character introduced - which we also prefer to not have in this repository... sorry for being picky again